R/multivar_match_helpers.R

```diff
@@ -31,7 +31,7 @@ compare_row <- function(row_x, data2,
   #------------------------
   ## generate comparisons
   #------------------------
-  # note that this requi
+  # note that this requires ??
```

Comment:
This isn't intended as a fix, it's a note that the current text doesn't make sense (it dates to its introduction).